### PR TITLE
added routes

### PR DESCRIPTION
**Pull request by polanpio**
I've added routes but I'm not sure if the two for dashboard and bookings are OK. Routes for pets seem fine. Double check, please ;)

### CODE REVIEW

**Review by ellie-xyb — APPROVED** [review]
LGTM(looks good to me)
You can merge this one whenever:) 
Seems like in the big company is the people who made the branch should merge pull request after getting the review.

**Comment by polanpio** [comment]
Ok. Is it like this? last time you merged seeds branch

File: config/routes.rb [hunk 1]
```diff
@@ -1,5 +1,10 @@
 Rails.application.routes.draw do
   devise_for :users
-  root to: 'pages#home'
+  root to: 'pets#home'
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  resources :pets do
+    resources :bookings, only: [:create]
+  end
+  get "dashbord", to: 'dashboards#display'
```
Comment (ellie-xyb) [inline config/routes.rb]:
```suggestion
  get "dashboard", to: 'dashboards#display'
```

**Review by ellie-xyb — CHANGES_REQUESTED** [review]

**Comment by ellie-xyb** [comment]
Sorry, just found a typo.

**Comment by ellie-xyb** [comment]
> Ok. Is it like this? last time you merged seeds branch

Yeah, last time I didn't know. Just got the advice today.

**Comment by polanpio** [comment]
Aaaaah ok! I messed up branching so there will doubled code in two branches. Let's see what will happen.

**Review by ellie-xyb — APPROVED** [review]

**Pull request merged**